Problematic Include usage in SMArtInt.Internal.ModelicaUtilityHelper

Per the newest specification: https://specification.modelica.org/master/functions.html#modelica:Include

> The Include annotation shall be used in such a way that each external function can be handled in a separate translation unit. In particular, different external functions must not have Include annotations providing exported definitions of the same function symbol to avoid linking errors.

But:

https://github.com/xrg-simulation/SMArtInt/blob/0f8ff42b6728533ae36ce8349827e5c6de4f0d96/SMArtInt/Internal/ModelicaUtilityHelper.mo#L8-L14

which results in this compilation error if a tool builds each function in a separate translation unit (this particular example from Wolfram System Modeler):

```
ld: exename_ext16-destructor.o: in function `createModelicaUtitlityHelper':
   exename_ext16-destructor.c:(.text+0x0): multiple definition of `createModelicaUtitlityHelper'; exename_ext15-constructor.o:exename_ext15-constructor.c:(.text+0x0): first defined here
ld: exename_ext16-destructor.o: in function `deleteModelicaUtitlityHelper':
   exename_ext16-destructor.c:(.text+0x29): multiple definition of `deleteModelicaUtitlityHelper'; exename_ext15-constructor.o:exename_ext15-constructor.c:(.text+0x29): first defined here
```

Here is the language specification discussion around this topic: https://github.com/modelica/ModelicaSpecification/issues/3451

--- a/SMArtInt/Resources/Include/ModelicaUtilityInterface.cpp
+++ b/SMArtInt/Resources/Include/ModelicaUtilityInterface.cpp
@@ -6,14 +6,14 @@
 
 static ModelicaUtilityHelper s_ModelicaUtilityHelper;
 
-void* createModelicaUtitlityHelper()
+static void* createModelicaUtitlityHelper()
 {
 	s_ModelicaUtilityHelper.ModelicaError = ModelicaError;
 	s_ModelicaUtilityHelper.ModelicaMessage = ModelicaMessage;
 	return &s_ModelicaUtilityHelper;
 }
 
-void deleteModelicaUtitlityHelper(void* externalObject)
+static void deleteModelicaUtitlityHelper(void* externalObject)
 {
 
 }

